### DISCUSSION TITLE
Theme doesn't applying in zsh

### DISCUSSION THREAD

**Discussion by shadyueh** [starship/starship#5461] [Q&A]
I've followed the installation guide to install starship in zsh on Ubuntu Jammy
But i noticed nothing changed in zsh terminal.

![image](https://github.com/starship/starship/assets/325680/2151bb98-ecba-45d8-a25e-6cebf70f6ac2)

![image](https://github.com/starship/starship/assets/325680/afa62d61-c8d3-429e-be10-d85f0346835d)

Is there any  step I'm missing?

I already rebooted the computer but still don't see the theme applied.

**Comment by thesocialdev** [comment]
It's sad this is not answered
